### DISCUSSION TITLE
SEB Proctoring deprecated

### DISCUSSION THREAD

**Discussion by Mark-723** [SafeExamBrowser/seb-server#134] [Q&A]
Recently, it was announced that the SEB Server Proctoring will be removed with the next version of SEB Server.
We where hoping to use the SEB Proctoring once it has left the experimantal status.

Does this mean that the proctoring function will not further developed at all? Or will it potentially come up again in the future?
What was the reason for this decision?
Thanks.

Mark

**Comment by anhefti** [comment]
Hi Mark

Yes, SEB Server Live Proctoring with Zoom and Jitsi Meet is marked as deprecated since version 1.6 and will be removed on the upcoming version 2.0. This is because of high maintenance cost in development as well as in production and support, and lack of demand from SEB Alliance. You are able to still use Live Proctoring with Zoom and Jitsi Meet with SEB Server version 1.6 but without support and guarantee that is works with newest Zoom or Jitsi Meet API. 

As a replacement, SEB Server 2.0 will have a new Screen Proctoring feature where it would be possible to take screenshots of SEB Clients and proctor them and also record them for later analysis. The SEB Server release is planned for sept. 2024.

Best
Andreas

**Reply by Mark-723** [reply]
Hi Andreas,
thank you for this additional information.

Best regards,
Mark
